builder/bundles.go

```diff
@@ -520,13 +520,22 @@ func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *b
 		return nil
 	}
 
-	metaPath := filepath.Join(baseDir, "usr/share/clear/allbundles")
-	err = os.MkdirAll(metaPath, 0755)
-	if err != nil {
-		return err
-	}
+	// Index manifest content is only generated for formats < 29 and when there
+	// is no bundle chroot.
+	_, err = os.Stat(filepath.Join(buildVersionDir, "os-core-update-index"))
+	if format <= "28" && os.IsNotExist(err) {
```

Comment:
The content for the os-core-update-index manifest is created during `mixer build bundles`. Should format switching in the builder package be centralized like swupd?

Reply:
At a glance, I think I should be able to consolidate this in one place.

The index manifest entries are generated from the same object that populates the bundle-info files. During `mixer build update`, the bundle-info files are loaded into the `BundleInfo` object. This could be used to generate the index manifest entries.